app/assets/javascripts/assignment_form.js

```diff
@@ -0,0 +1,7 @@
+function import_options(starter_code_field_value) {
```

Comment:
Is snake_case used elsewhere in the app? If not, could go with camelCase to match JavaScript style, for both the variable and the function name.

Reply:
Good catch! Looks like we've used both in our `assets/javascripts` but I'll switch to camelCase here.